package.json

```diff
@@ -19,9 +19,11 @@
     "stylus": "^0.54.5"
   },
   "devDependencies": {
+    "@google-cloud/datastore": "^0.3.1",
```

Comment:
I believe you can use "@gcloud/datastore"

I do not believe we need to use both google-cloud and gcloud. One supersedes the other... I think